macros/sql/get_query_results_as_single_value.sql

```diff
@@ -0,0 +1,29 @@
+{% macro get_query_results_as_single_value(query, row_position=0, column_position=0) %}
```

Comment:
allowing custom row and column positions is interesting! I guess I'm not against it for column position, but it also feels a lil bit more complicated than is maybe necessary. Is there a use case you had in mind (other than making the integration tests a bit tidier)?

If we do _go_ down this route, it makes sense for column position but probably not row position. I can't imagine a situation where I want the third row's data for this sort of query - it should be returning a single row only.

Reply:
I've had a change of heart, I don't think these are necessary. They felt useful at the time, but I can't actually think of a time that I'd give this macro a query, but not specify exactly which column/row I was looking for in the query itself. Removing this!